src/sentry/web/api.py

```diff
@@ -267,7 +267,17 @@ class StoreView(APIView):
 
     """
     def post(self, request, **kwargs):
-        data = request.body
+        try:
+            data = request.body
+        except Exception:
```

Comment:
Unfortunately not: https://github.com/django/django/blob/1.6.11/django/http/request.py#L186

I can add logging though.